vendor/bat-native-ads/src/bat/ads/internal/account/user_data/conversion_user_data.cc

```diff
@@ -5,27 +5,33 @@
 
 #include "bat/ads/internal/account/user_data/conversion_user_data.h"
 
+#include <utility>
+
 #include "base/check_op.h"
+#include "base/functional/callback.h"
 #include "bat/ads/confirmation_type.h"
 #include "bat/ads/internal/account/user_data/conversion_user_data_builder.h"
 
 namespace ads::user_data {
 
 void GetConversion(const std::string& creative_instance_id,
                    const ConfirmationType& confirmation_type,
-                   const ConversionCallback& callback) {
+                   ConversionCallback callback) {
   DCHECK(!creative_instance_id.empty());
   DCHECK_NE(ConfirmationType::kUndefined, confirmation_type.value());
 
   if (confirmation_type != ConfirmationType::kConversion) {
-    callback(base::Value::Dict());
+    std::move(callback).Run(base::Value::Dict());
     return;
   }
 
-  builder::BuildConversion(creative_instance_id,
-                           [callback](const base::Value::Dict& user_data) {
-                             callback(user_data.Clone());
-                           });
+  builder::BuildConversion(
+      creative_instance_id,
+      base::BindOnce(
+          [](ConversionCallback callback, base::Value::Dict user_data) {
+            std::move(callback).Run(user_data.Clone());
```

Comment:
you now have `base::Value::Dict user_data` here, so you don't need to `Clone()`.
`Run(std::move(user_data))`

Reply:
Fixed